haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
So then in that case I could log something like `Warning: sleep_time greater than 10 minutes` if someone set `sleep_time` to, say, greater than 600 seconds, but the program won't exit.  What do you think @henn ?